src/index.js

```diff
@@ -1,23 +1,43 @@
 /* eslint-disable no-plusplus */
 import './css/style.css';
-// import NewRecord from './new-record.js';
+import NewRecord from './new-record.js';
 import RecentScores from './recent-scores.js';
 
-function getRecentScores() {
-  const recentScoresList = document.getElementById('recent-scores-list');
-  const ul = document.createElement('ul');
-  const recentScores = RecentScores.getRecentScores();
-  for (let i = 0; i < recentScores.length; i++) {
-    const li = document.createElement('li');
-    li.appendChild(
-      document.createTextNode(
-        `${recentScores[i].name}: ${recentScores[i].score}`,
-      ),
-    );
-    ul.appendChild(li);
-  }
+const recentScoresList = document.getElementById('recent-scores-list');
 
-  recentScoresList.appendChild(ul);
+function getRecentScores() {
```

Comment:
- [ ] Your function does not conform to the _arrow functions_ syntax. Kindly change the _normal_ function to an _arrow function_ syntax, so as to keep your _function_ syntax simpler and cleaner

Reply:
This issue is well addressed. Thank you for making the requested changes. :clap: